### ISSUE TITLE
1008p looks more blurry than 900p

### ISSUE THREAD

**Issue by Foul-Tarnished** [HolographicWings/TOTK-Mods-collection#1]
with disable dynamic res and disable fsr mods

![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/107711110/730c386c-69dd-44f1-a37c-344513f5d8a3)
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/107711110/c83ac59b-4b17-4fcb-a0d1-97f262e1902b)

Link, the trees and his horse, all looks more blurry, less sharp

mods i'm using : 
```
//Disable Variable Resolution Scaling
@enabled
00cea77c 15000014
@stop

// FSR (Pretty much just skipping the entire shader? (I could be wrong))
@enabled
027CA0DC 000080D2
0188D0D4 000080D2
00AEAC44 1F2003D5
@stop 

// FXAA (Probably does nothing, but makes sure FXAA is not enabled)
@enabled
027CA13C 000080D2
0188D0A0 000080D2
@stop

// patchanon Disable FSR downscaling
@enabled
00CAD348 08008052
@stop

// patchanon Disable quality reduction when framerate dips
@enabled
00CAD34C C0000014
@stop
```

**Comment by HolographicWings** [comment]
I'm sorry i don't have the eyes for this slight differences, i'll ask to Hover to investigate
the only difference i see is on the horse's mane yes that's true

**Comment by Foul-Tarnished** [comment]
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/107711110/66e40298-9a03-44f9-8e40-9264e53cad48)
How can you not see ?
It's slightly less aliased, but it's way more blurry

**Comment by HolographicWings** [comment]
> ![image](https://user-images.githubusercontent.com/107711110/238196510-66e40298-9a03-44f9-8e40-9264e53cad48.png) How can you not see ? It's slightly less aliased, but it's way more blurry

that's true, did you tried x2 resolution scale in the yuzu settings ? the performances lacks are from CPU and not GPU, you should try

**Comment by Foul-Tarnished** [comment]
I loose too much fps with 2x.
Also 2x makes it blurry with 900p, and shimmery with 1008p, so meh

Also I use Ryuinx :) way better.

**Comment by hoverbike1** [comment]
1008 is not divisible by 1080 (in a whole number)
This means that scaling it to a 1080p viewport becomes blurry. This could possibly be fixed by using a 1008p sized windowed viewport, but I am not aware of any way to set windows to a certain height or width. 
If anyone knows such software, it may be able to solve this issue, albeit by using a smaller viewport.

Now, to solve your specific issue, I think we have to get a bit more into opinion over fact, since the presentation of an image, is in some part, defined by user preference.

You should try the old, deprecated 1080p patch. It may be to your liking. It has great performance, and looks like this at x1:
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/57139770/2068d8be-da97-44a2-abf8-f540dd524e6d)
If you would like further help to tailor your experience of TOTK to your personal preference, feel free to hit us up on the Yuzu discord, in the game-mods channel, but do keep in mind, that we can be quite busy at times.
We want to help though, so please be patient.

**Comment by Foul-Tarnished** [comment]
I don't use shitty emulators, sorry.
